app/templates/calendar_day_view.html

```diff
@@ -34,6 +34,14 @@
           </div>
         {% endfor %}
     </div>
+    <div class="timegrid">
+      {% if current_time.is_viewed %}
+        <div class="sub-timegrid" style="grid-row: {{ current_time.grid_position }};">
+            <div id="current_time_cursor" style="grid-row: {{ current_time.sub_grid_position}};">
```

Comment:
Prefer putting the style in the CSS

Reply:
Mmm but it gets changing parameters for the grid-row according to current time